external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/KafkaSpout.java

```diff
@@ -125,8 +125,8 @@ public void open(Map<String, Object> conf, TopologyContext context, SpoutOutputC
 
         tupleListener = kafkaSpoutConfig.getTupleListener();
 
-        if (isAtLeastOnce()) {
-            // Only used if the spout commits offsets for acked tuples
+        if (isAtLeastOnceProcessing()) {
+            // Only used if the spout should commit to Kafka an offset only after its tuple has been acked.
```

Comment:
"Should commit an offset to Kafka only after the corresponding tuple has been acked"

Reply:
Done.